app/components/trainees/route_indicator/view.rb

```diff
@@ -15,8 +15,34 @@ def render?
         trainee.training_route.present? && trainee.draft?
       end
 
+      def display_text
+        if trainee.apply_application?
+          "Trainee enrolled on #{course_with_code}, #{training_route}"
+        else
+          "Trainee on the #{training_route_link} route".html_safe
+        end
+      end
+
+      def course_with_code
+        "#{trainee.subject}#{course_code}"
+      end
+
+      def course_code
+        return unless course&.code
+
+        " (#{course.code})"
+      end
+
+      def course
+        @course ||= trainee.provider.courses.find_by(name: trainee.subject)
```

Comment:
This will not work

![image](https://user-images.githubusercontent.com/470137/117992058-87083500-b336-11eb-992f-cf59aced32f4.png)

https://www.find-postgraduate-teacher-training.service.gov.uk/results?l=2&subjects%5B%5D=31

Reply:
Add new column `course_code` on trainee, updated `CreateFromApply` to save the value from Apply